src/ui/components/Library/Team/View/TestRuns/BranchIcon.tsx

```diff
@@ -8,7 +8,7 @@ export function BranchIcon({
   title,
 }: {
   branchName: ReactNode;
-  isPrimaryBranch: boolean;
+  isPrimaryBranch: boolean | null;
```

Comment:
It was fallout from an [earlier backend change](https://github.com/replayio/backend/pull/8775) but wasn't reflected in devtools because the graphql types weren't updated.